src/app/store/actions/VehicleTestResultModel.actions.spec.ts

```diff
@@ -1,36 +1,15 @@
 import {
   EVehicleTestResultModelActions, GetVehicleTestResultModel, GetVehicleTestResultModelSuccess, GetVehicleTestResultModelFailure
 } from './VehicleTestResultModel.actions';
-import { TestResultModel } from '@app/models/test-result.model';
+import {TestResultModel} from '@app/models/test-result.model';
 
-const testResultModelExample: TestResultModel = {
-  testerStaffId: '21',
-  testStartTimestamp: new Date(),
-  odometerReadingUnits: 'test',
-  testEndTimestamp: new Date(),
-  testStatus: 'new',
-  testTypes: [],
-  vehicleClass: null,
-  testStationName: 'test',
-  vehicleId: '213',
-  noOfAxles: 2,
-  vehicleType: 'none',
-  countryOfRegistration: 'Ro',
-  preparerId: '231',
-  odometerReading: 123,
-  vehicleConfiguration: 'none',
-  testStationType: 'none',
-  reasonForCancellation: 'none',
-  testerName: '213',
+const testResultModelExample = {
```

Comment:
As the code in place was prior to ADR, i have added a technical task ticket to refactor the state names for test result as well as we discussed and agreed yesterday.
Please let me know if you don't agree.

https://jira.dvsacloud.uk/browse/CVSB-15127